examples/header.php

```diff
@@ -8,3 +8,6 @@
 
 $api_key = getenv('API_KEY');
 $reference_id = getenv('REFERENCE_ID');
+$scheme = 'https';
```

Comment:
After correcting the `__contstructor` we can remove all these extra params - it should work by default then.